py/objfloat.c

```diff
@@ -52,11 +52,13 @@ typedef struct _mp_obj_float_t {
     mp_float_t value;
 } mp_obj_float_t;
 
-const mp_obj_float_t mp_const_float_e_obj = {{&mp_type_float}, M_E};
-const mp_obj_float_t mp_const_float_pi_obj = {{&mp_type_float}, M_PI};
+const mp_obj_float_t mp_const_float_e_obj = {{&mp_type_float}, (mp_float_t)M_E };
+const mp_obj_float_t mp_const_float_pi_obj = {{&mp_type_float}, (mp_float_t)M_PI };
```

Comment:
The spaces before the `}` here should probably go.

Reply:
I'll check if Uncrustify can do that, likewsie for the spaces after `//`

Reply:
The option `sp_cmt_cpp_start` will do spaces after the comment, but you'll see that it changes also commented out code.... but I guess that's just a good excuse to remove commented out code!